modules/cucumber_testsuite/main.tf

```
@@ -70,6 +72,7 @@ module "pxy" {
 
   base_configuration = module.base.configuration
   product_version    = var.product_version
+  image              = lookup(local.images, "pxy", null)
```

If the image is not defined we should use the default value for the variable which is: "default".
```suggestion
  image              = lookup(local.images, "pxy", "default")
```